fluent-api/src/test/java/org/codice/imaging/nitf/fluent/CreationFlowTest.java

```diff
@@ -58,9 +63,11 @@ public CreationFlowTest() {
     @Test
     public void createEmptyNitf() throws FileNotFoundException, NitfFormatException {
         final String filename = "empty.ntf";
-        new NitfCreationFlow()
+        DataSource dataSource = new NitfCreationFlow()
                 .fileHeader(() -> NitfHeaderFactory.getDefault(FileType.NITF_TWO_ONE))
-                .write(filename);
+                .build();
+        new NitfFileWriter(dataSource, filename).write();
+
```

Comment:
This looks worse than what we had previously. It would be nice to have an API that is fully streaming.